ShanghaiBloodSports/Assets/Scripts/Character.cs

```diff
@@ -1,137 +1,171 @@
-﻿using Assets.Scripts.Moves;

```

Comment:
🤢

Reply:
Did this file change or did git just die

Reply:
![image](https://user-images.githubusercontent.com/6673570/54799126-701b9b80-4c21-11e9-9348-34c14d01e81f.png)

Reply:
I mean, yes, it was git's fault

Reply:
And also, yes, the file was actually changed